src/main/scala/org/allenai/deep_qa/data/JohannesDatasetReader.scala

```diff
@@ -0,0 +1,31 @@
+package org.allenai.deep_qa.data
+
+import scala.collection.mutable
+
+import com.mattg.util.FileUtil
+import org.json4s._
+import org.json4s.Formats
+import org.json4s.native.JsonMethods.parse
+
+class JohannesDatasetReader(fileUtil: FileUtil) extends DatasetReader[MCReadingComprehensionInstance] {
+  override def readFile(filename: String): Dataset[MCReadingComprehensionInstance] = {
+    val json = parse(fileUtil.readFileContents(filename))
+    val instanceTuples = for {
+      JObject(mcQuestion) <- json
+      JField("support", JString(passage)) <- mcQuestion
+      JField("question", JString(question)) <- mcQuestion
+      JField("distractor1", JString(distractor1)) <- mcQuestion
+      JField("distractor2", JString(distractor2)) <- mcQuestion
+      JField("distractor3", JString(distractor3)) <- mcQuestion
+      JField("correct_answer", JString(correctAnswer)) <- mcQuestion
+    } yield (passage, question, distractor1, distractor2, distractor3, correctAnswer)
+
+    val instances = instanceTuples.map { case (passage, question, distractor1,
+      distractor2, distractor3, correct_answer) => {
+      val options = Seq(distractor1, distractor2, distractor3, correct_answer).sorted
+      val label = options.indexOf(correct_answer)
```

Comment:
You need to replace tabs with spaces in the answer options in here somewhere.  Somehow there are some tabs in the data.